src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -234,19 +244,20 @@ class ClangTidyIssue(Issue):
     '''
     ANALYZER = CLANG_TIDY
 
-    def __init__(self, header_data, revision):
-        assert isinstance(header_data, tuple)
-        assert len(header_data) == 6
+    def __init__(self, revision, path, line, char, check, message, type='warning'):
         assert not settings.repo_dir.endswith('/')
         self.revision = revision
-        self.path, self.line, self.char, self.type, self.message, self.check = header_data  # noqa
+        self.path = path
         if self.path.startswith(settings.repo_dir):
             self.path = self.path[len(settings.repo_dir)+1:]  # skip heading /
-        self.line = int(self.line)
+        self.line = int(line)
         self.nb_lines = 1  # Only 1 line affected on clang-tidy
-        self.char = int(self.char)
+        self.char = int(char)
+        self.check = check
+        self.message = message
         self.body = None
         self.notes = []
+        self.type = type
```

Comment:
Nit: maybe use a different variable name since `type` is a keyword

Reply:
Updated the local usage, and created a [good-first-bug issue](https://github.com/mozilla/release-services/issues/1980) to do it everywhere